com.gluonhq.scenebuilder.yml

```
@@ -0,0 +1,63 @@
+app-id: com.gluonhq.scenebuilder
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: SceneBuilder
+finish-args:
+  - --socket=x11
```

```suggestion
  - --socket=x11
  - --share=ipc
```

This is needed for X11.